src/current/cockroachcloud/cluster-management-advanced.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Manage a CockroachDB Advanced Cluster
```

Comment:
i'm explicitly ignoring this and looking at the link stuff!  Trust that this is content in your area that if you need a real review you can/will get as needed